### PR TITLE
1183 invalid csv missing error message 

### LINKED ISSUE #1180: Unexpected error while loading file
- Catch and present properly
- Add a fullstop at the end of the message.

### LINKED ISSUE #1183: Importing this csv does not show why it is not valid
[sample1.zip](https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/files/6821565/sample1.zip)

### PR DESCRIPTION
**Pull request by georgeDaskalakis**
closes #1180 
closes #1183

### CODE REVIEW

File: src/OSPSuite.Assets/UIConstants.cs [hunk 1]
```diff
@@ -1262,11 +1262,24 @@ public static class Error
       public static readonly string NoResultsAvailableForExportToCSV = "No results available for export to CSV";
       public static readonly string NamingConventionEmpty = "Column naming conventions cannot be empty.";
       public static readonly string NamingConventionNull = "Column naming conventions cannot be null.";
-      public static readonly string InvalidFileException = "An error occurred while reading the file. Please check the content";
       public static readonly string InvalidMappingColumn = "An invalid mapping column has been used. Check your import configuration and data file";
       public static readonly string InvalidErrorDimension = "The dimension of the error units must be the same as the dimension of the measurement units.";
       public static readonly string NaNOnData = "Data contains NaN values at imported columns. Select a different action for NaN values or clean your data.";
       public static readonly string UnsupportedFileType = "The type of file that you are trying to open is not currently supported";
+
+      public static string InvalidFileException(string exceptionMessage)
```
Comment (Yuri05) [inline src/OSPSuite.Assets/UIConstants.cs:1270]: Let's rename to `InvalidObservedDataFile`

File: src/OSPSuite.Infrastructure.Import/Core/InvalidFileException.cs [hunk 2]
```diff
@@ -1,11 +1,12 @@
 ﻿using OSPSuite.Assets;
 using System;
+using OSPSuite.Utility.Extensions;
 
 namespace OSPSuite.Infrastructure.Import.Core
 {
    public class InvalidFileException : Exception
```
Comment (Yuri05) [inline src/OSPSuite.Infrastructure.Import/Core/InvalidFileException.cs:7]: please derive from `OSPSuiteException`

Comment (Yuri05) [reply src/OSPSuite.Infrastructure.Import/Core/InvalidFileException.cs:7]: also other importer exceptions should be derived from OSPSuiteException
grafik

File: src/OSPSuite.Assets/UIConstants.cs (hunk 1 above)
Comment (georgeDaskalakis) [reply src/OSPSuite.Assets/UIConstants.cs:1270]: done. renamed also the exception accordingly.

File: src/OSPSuite.Infrastructure.Import/Core/InvalidFileException.cs (hunk 2 above)
Comment (georgeDaskalakis) [reply src/OSPSuite.Infrastructure.Import/Core/InvalidFileException.cs:7]: done. I also realized there are 3 more exceptions that derive directly from Exception. Should I change those too, or are they on purpose?
image

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**